quarg.py

```diff
@@ -73,6 +75,11 @@ def parse_docstring(doc):
     else:
         return ('', '', '')
 
+def envstring_to_bool(value):
+    """Convert an environment variable to a boolean"""
+    return (False if value is None
```

Comment:
Necessary? I think it's more accepted just to check whether it's set or not rather than an opaque false-ish string.

Reply:
I was unsure of this; I agree it's messy, but I think it's useful to have a way to positively assert that the flag is false. How about enabling the mode if the environment variable is set to a _non-empty_ value (i.e., `bool(os.getenv('QUARG_DEBUG'))`?

Reply:
That seems like a good solution